### ISSUE TITLE
Spinner will still write to STDOUT even when set to STDERR

### ISSUE THREAD

**Issue by spaghetti-is-pasta** [oclif/cli-ux#132]
Even if you select STDERR for the spinner, using:

`cli.action.start(msg, null, { stdout: false })`

the spinner will still write line-feeds to STDOUT. For utilities that a require full control over STDOUT (think `gz`, or `tar` or similar) this is unacceptable as it corrupts the output. The issue seems to be in:

https://github.com/oclif/cli-ux/blob/06ef95fe483e6c19c53f6b4273e24d4b30432f99/src/action/base.ts#L175

`std` appears to get set to `"stdout"` somehow.

**Comment by RasPhilCo** [comment]
Good catch, this need to be fixed. I'm out tomorrow if you can work up a patch, otherwise I'll get to it next week.
